src/TurboLinks.Net/TurboLinks.Net.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Add support for Turbolinks</Description>
+    <VersionPrefix>1.1.1</VersionPrefix>
+    <Authors>Tommy Parnell</Authors>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I'm not going to drop support for netstandard1.6. Please target both. I am not going to introduce a breaking change such as this.